guides/common/modules/proc_adding-proxmox-details-to-a-compute-profile.adoc

```diff
@@ -12,34 +12,15 @@ Note that this procedure assumes you are using KVM/qemu-based virtualization.
 . Select a Proxmox compute resource.
 // FIXME: this procedure currently assumes you've selected KVM/qemu
 . In the *Type* list, select the kind of virtualization solution hosts will run on when created using this compute profile.
-In Proxmox, you may choose between _LXC_ (Linux containers) and _KVM/QEMU_ (kernel-based virtual machines).
+In Proxmox, you can choose between *LXC container* (Linux containers) and *KVM/Qemu server* (kernel-based virtual machines).
 LXC focuses more on isolating applications and reusing the host kernel.
 KVM runs its own kernel and lets you boot various guest operating systems.
+. Click on the *General* tab.
 . In the *Node* list, select the Proxmox node you want to deploy your hosts to.
 This makes sense if you run a Proxmox cluster.
```

Comment:
Is this really in the documentation? Seems unnecessary. :-)

Reply:
Dropped.